src/componentsEventPage/CategoryFilter.js

```diff
@@ -9,10 +9,6 @@ export default function CategoryFilter({ onSelectFilter }) {
     <FilterContainer>
       <SelectCategory id="category" name={["metal", "medieval", "sand", "other"]} 
         options={["metal", "medieval", "sand", "other"]}  onSelectFilter={onSelectFilter}/>
```

Comment:
is name and option always the same? then you could use a uppercase const variable to safe the list there and use it in line 10 and 11

```suggestion
const OPTIONS = ["metal", "medieval", "sand", "other"]
<FilterContainer>
      <SelectCategory id="category" name={OPTIONS} 
        options={OPTIONS}  onSelectFilter={onSelectFilter}/>
```

Reply:
The Select component needs an options and a name, otherwise, it's not working. These are two different props.